lib/src/lightclient.rs

```diff
@@ -1559,8 +1584,20 @@ impl LightClient {
         result.map(|(transaction_id, _)| transaction_id)
     }
 
+    async fn latest_served_blockheight(&self) -> BlockHeight {
+        let latest_height = GrpcConnector::get_latest_block(self.config.get_server_uri())
+            .await
+            .unwrap()
+            .height;
+        BlockHeight::from_u32((latest_height - self.config.reorg_buffer_offset as u64) as u32)
```

Comment:
This function's name mislead me into think this function returns the latest server blockheight, not the latest served height minus our intended anchor offset.

Reply:
I think the function is named what it wants to do, and this `-` bit should be removed.